packages/core/js-sdk/src/admin/order.ts

```diff
@@ -0,0 +1,81 @@
+import {
+  AdminCancelOrderFulfillment,
+  FindParams,
+  HttpTypes,
+  PaginatedResponse,
+  SelectParams,
+} from "@medusajs/types"
+import { Client } from "../client"
+import { ClientHeaders } from "../types"
+
+export class Order {
```

Comment:
Nice, think we should create types for this, but to not extend the scope of the PR further, we could save that for a follow-up PR